boards/arm/stm32_blackpill_v2/doc/index.rst

```diff
@@ -0,0 +1,163 @@
+.. _stm32_blackpill_v2:
+
+WeAct Studios Black Pill V2.0
```

Comment:
It seems that the version 3 is just a bugfix version on hardware level, so this code should also run on V3.0, see https://github.com/WeActTC/MiniF4-STM32F4x1/blob/master/HDK/README.md regarding the changes. Maybe this could be mentioned in this doc.

Edit: I can test this on the V3 board in the next days.

Reply:
Thanks for the feedback! And a test is very much appreciated. I will look into this and try to make it clear which versions can be used with this board.

Reply:
> It seems that the version 3 is just a bugfix version on hardware level, so this code should also run on V3.0, see https://github.com/WeActTC/MiniF4-STM32F4x1/blob/master/HDK/README.md regarding the changes. Maybe this could be mentioned in this doc.
> 
> Edit: I can test this on the V3 board in the next days.

So I used your input combined with erwango's and have updated some things. I removed the specificity of V2.0 from the docs/board definition completely, and instead went with "V2.0+" where possible, as this is what WeAct does in their documentation where appropriate it looks like. I changed the board name completely to blackpill_f411ce. This should be clearer regarding exactly which board this is, because there are lots of pill boards.

The revision history was a great resource, so thanks for that, because I did learn that new info about MISO. I switched to using PA6 as the default pinmux for MISO, and made a note in the docs and in the code for pinmux.c regarding that possible issue.